verifier/verifier.go

```diff
@@ -212,6 +217,18 @@ func (v *Verifier) Verify(c *x509.Certificate, opts VerificationOptions) (res *V
 		res.Parents = parentsFromChains(res.CurrentChains)
 	}
 
+	if opts.OneCRL != nil && opts.OneCRL.Check(c) != nil {
+		res.InRevocationSet = true
+	}
+	if opts.CRLSet != nil {
```

Comment:
I don't know how expensive these checks will end up being, but I'm guessing not super cheap, so maybe we should use `else if` since theres nothing to be gained from doing both branches

```suggestion
	else if opts.CRLSet != nil {
```

Reply:
`else` is not appropriate here.
`CRLSet` and `OneCRL` are different source of info. and it's already in memory. no penalty on the check.

Reply:
Sure but we don't get anything out of doing the second check if the first branch was true because they set the same bool?

Reply:
@bwireman instead of `else` I added this condition `!res.InRevocationSet` 

`if !res.InRevocationSet && opts.CRLSet != nil {`